operations/loyaltymemberships.md

```diff
@@ -0,0 +1,280 @@
+# Loyalty memberships
+
+## Get all loyalty memberships
+
+Returns all loyalty memberships of the enterprise (in the given activity states), optionally filtered by specific loyalty membership identifiers or other filter parameters.
+Note this operation uses [Pagination](../guidelines/pagination.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/loyaltyMemberships/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0"
+    "LoyaltyMembershipIds": [
+        "3f4d9db2-9910-4a63-b9f0-e94a13fab9ac",
+        "ea7da00f-fdc9-4014-b0f7-71003b87e3d0"
+    ],
+    "LoyaltyProgramIds": [
+        "3ed9e2f3-4bba-4df6-8d41-ab1b009b6425",
+        "8a98965a-7c03-48a1-a28c-ab1b009b53c8"
+    ],
+    "AccountIds": [
+        "87d4c7c4-4832-4341-8b54-e45c1a73df34",
+        "0ed43ab7-4592-4c99-906a-426588de1c00"
+    ],
+    "CreatedUtc": {
+        "StartUtc": "2022-10-15T00:00:00Z",
+        "EndUtc": "2022-10-20T00:00:00Z"
+    },
+    "UpdatedUtc": {
+        "StartUtc": "2022-10-15T00:00:00Z",
+        "EndUtc": "2022-10-20T00:00:00Z"
+    },
+    "ActivityStates": [
+        "Active"
+    ],
+    "Limitation":{
+        "Cursor": "ea7da00f-fdc9-4014-b0f7-71003b87e3d0",
+        "Count": 100
+    }
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `LoyaltyMembershipIds` | array of string | optional, max 1000 items | Unique identifiers of [Loyalty memberships](#loyalty-membership). |
+| `LoyaltyProgramIds` | array of string | optional, max 1000 items | Unique identifiers of [Loyalty programs](loyaltyprograms.md#loyalty-program). |
+| `AccountIds` | array of string | optional, max 1000 items | Unique identifiers of [Customers](customers.md#customer) or [Companies](companies.md#company). |
```

Comment:
We have it phrased like this in other endpoints. I think it more clearly says that the Customer and Company are basically same thing.
```suggestion
| `AccountIds` | array of string | optional, max 1000 items | Unique identifiers of accounts (for example [Customers](customers.md#customer) or [Companies](companies.md#company)) the membership is associated with. |
```